recipes/transannot/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3-70b2a60" %}
-{% set sha256 = "3b1d2f28362004378528755b8b4e0b8dc45b51cc60562816258bdf16d9cda08d" %}
+{% set version = "3-e15e316" %}
```

Comment:
@mariia-zelenskaia is this a hash in your version? Please note that if you do that there is no guarantee that one version is bigger than another. So you end up installing an old version by default.